src/activities/instanceCollections/CollectedObject.js

```diff
@@ -0,0 +1,22 @@
+import { Entity } from '../../es6/Entity.js';
+
+export class CollectedObjectEntity extends Entity {
+	getSortOrder() {
+		if (!this._entity) {
+			return false;
+		}
+		return this._entity.properties?.SortOrder;
+	}
+	getCompletion() {
```

Comment:
Is this a "completion type"? Maybe thats a better name? I wasnt sure what completion meant and had to look at the entity below.

Reply:
right! Okay, I am going to note this comment, we are still going to be adding a ton here , so we can update this

Reply:
I was also debating if we even need to put "get" , maybe just "completionType" is enough? same can be applied to the other functions